debug/FileSaver.js

```diff
@@ -30,7 +30,7 @@
     if (typeof opts === 'undefined') opts = {
       autoBom: false
     };else if (typeof opts !== 'object') {
-      console.warn('Depricated: Expected third argument to be a object');
+      console.warn('Deprecated: Expected third argument to be a object');
```

Comment:
This file is just copied from here: https://github.com/eligrey/FileSaver.js/

I suppose they have, or had, a typo in their repo.  No harm in changing that here, but the file is only used for debugging anyway.  Users would never see it.